README.md

```diff
@@ -1,42 +1,65 @@
 # alexa-app
 
+A Node module to simplify the development of Alexa skills (applications.)
+
 [![NPM](https://img.shields.io/npm/v/alexa-app.svg)](https://www.npmjs.com/package/alexa-app/)
 [![Build Status](https://travis-ci.org/alexa-js/alexa-app.svg?branch=master)](https://travis-ci.org/alexa-js/alexa-app)
 [![Coverage Status](https://coveralls.io/repos/github/alexa-js/alexa-app/badge.svg?branch=master)](https://coveralls.io/github/alexa-js/alexa-app?branch=master)
 
-A Node module to simplify development of Alexa apps (Skills) using Node.js.
+This module parses HTTP JSON requests from the Alexa platform and builds the JSON response that consumed by an Alexa-compatible device, such as the Echo.
 
-# Installation
+It provides a DSL for defining intents, convenience methods to more easily build the response, handle session objects, and add cards.
+
+The intent schema definition and sample utterances are included in your application's definition, making it very simple to generate hundreds (or thousands!) of sample utterances with a few lines.
+
+This module provides a way to host a standalone web service for an Alexa skill. If you're looking for a full-fledged application server 
+or the ability to host multiple skills, check out [alexa-app-server](https://github.com/alexa-js/alexa-app-server).
+
+
+### Features
+
+- simplified handling of requests and generating responses
+- support for asynchronous handlers
+- easy connection into AWS Lambda or Node.js Express, etc.
+- auto-generation of intent schema and sample utterances
+- support for session data
+- comprehensive test suite
 
-```bash
-npm install alexa-app --save
-```
 
-Node.js 0.12 or later is required.
+### AWS Lambda Example
 
-# Stable Release
+Amazon skills that use alexa-app have a built-in `handler` method to handle calls from AWS Lambda.
+You need to make sure that the Handler is set to `index.handler`, which is the default value.
 
-You're reading the documentation for the next release of alexa-app. Please see [CHANGELOG](CHANGELOG.md) and make sure to read [UPGRADING](UPGRADING.md) when upgrading from a previous version. The current stable release is [3.0.0](https://github.com/alexa-js/alexa-app/tree/v3.0.0).
```

Comment:
I'd like this to be put back. It links to a document with all the changes and makes it easy to find upgrading notes that are essential for anyone upgrading to a newer version.